oriedita/src/main/java/oriedita/editor/swing/LeftPanel.java

```diff
@@ -243,6 +244,7 @@ public LeftPanel(@Named("mainFrame") JFrame frame,
         buttonService.registerButton(correctCpBeforeFoldingCheckBox, "correctCpBeforeFoldingAction");
         buttonService.registerButton(selectPersistentCheckBox, "selectPersistentAction");
         buttonService.registerButton(coloredXRayCheckBox, "coloredXRayAction");
+        buttonService.registerButton(drawDiagonalGridlinesCheckBox, "drawDiagonalGridlinesAction");
```

Comment:
Please add entries for `drawDiagonalGridlinesAction` to `name.properties` and `help.properties`

Reply:
So when I added these, they did not directly show up in help or as a tooltip... Am I doing something trivially wrong?